packages/get-credentials/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@commercetools/get-credentials",
+  "version": "1.0.0",
+  "description": "Get CT credentials from the system environment.",
+  "keywords": [
+    "commercetools",
+    "credentials"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/commercetools/nodejs/tree/master/packages/get-credentials"
+  },
+  "homepage": "https://commercetools.github.io/nodejs/sdk/api/getCredentials.html",
+  "bugs": "https://github.com/commercetools/get-credentials/issues",
+  "license": "MIT",
+  "contributors": [
+    "Selwyn Versteeg <selwyn.versteeg@commercetools.com> (https://selwyn.cc/)"
+  ],
+  "main": "lib/index.js",
+  "files": [
+    "lib"
+  ],
+  "devDependencies": {
+    "sinon": "^1.17.7"
```

Comment:
The reason I went with a sandbox is that it doesn't influence the other tests. Doing it without a sandbox is a bit dangerous because one test can influence the result of others. Also relying on `process.env` without a sandbox means that developer's environment variables could collide with the tests.